documentation/docs/setting-up/server/easy-install.md

```diff
@@ -6,24 +6,24 @@
 ## Linux or macOS
```

Comment:
Do we also need to change the text in the note above? 
"You can download and check get-pmm2.sh before running it from our [github]"

Reply:
Maybe not... I believe it's still named [get-[mm.sh](https://github.com/percona/pmm/blob/main/get-pmm.sh), no?